microprofile/microprofile-fault-tolerance-2.0/src/main/resources/project-defaults-async.yml

```diff
@@ -0,0 +1,4 @@
+swarm:
+  microprofile:
+    fault-tolerance:
+      synchronous-circuit-breaker: false
```

Comment:
Why?

Reply:
obsolete, and should not be there at all